desc/objectives/_neoclassical.py

```diff
@@ -0,0 +1,253 @@
+"""Objectives for targeting neoclassical transport."""
+
+import numpy as np
+
+from desc.compute import get_profiles, get_transforms
+from desc.compute.utils import _compute as compute_fun
+from desc.grid import QuadratureGrid
+from desc.utils import Timer
+
+from ..backend import jnp
+from ..profiles import SplineProfile
+from .objective_funs import _Objective
+from .utils import _parse_callable_target_bounds
+
+
+class EffectiveRipple(_Objective):
+    """The effective ripple is a proxy for neoclassical transport.
+
+    Evaluation of 1/ν neoclassical transport in stellarators.
```

Comment:
Put this in a `References` block below, and maybe add another sentence or two describibing epsilon effective here

Reply:
eg, should mention that you're targeting it as a function of rho

Reply:
in objective.compute I had updated to write "Effective ripple as a function of the flux surface label"; is that sufficient